tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -352,6 +354,8 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 		val stepLog = step.contents.restoreString
 		logger.debug("generating code line for test step='{}'.", stepLog)
 		val interaction = step.interaction
+		 val testCase=EcoreUtil2.getContainerOfType(step, TestCase)
```

Comment:
This works only for test cases, not for macros, not for configurartions! Make sure that macros and configs are handled, too.